_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "coSMicQC: A step toward quality control and improvement of morphology profiles"
+author: Jenna Tomkinson
+member: jenna-tomkinson
+tags:
+  - opinions
+  - process
+  - software
+---
+# Let's tour the single-cell morphological galaxy with coSMicQC!
+
+Hi everyone! 🤩
+
+Now that I have a bit of bandwidth, I wanted to take the time to come back to go over something that I am very excited and passionate about:
+
+✨ *Quality control of single-cell morphology profiles* ✨
+
+In this blog post, I will go over what is quality control for morphology profiles, what current work exists on this topic, the software (coSMicQC) that we have developed to help in this topic, and where we hope to go from here.
+
+## Contents
+
+1. 📋 [What is quality control for single-cell morphology profiles?](#defining-quality-control-of-single-cell-morphology-profiles)
+2. 📉 [What current exists for this topic?](#what-quality-control-methods-currently-exist)
+3. 🌌 [Intro to coSMicQC](#introduction-to-cosmicqc)
+4. 🎯 [Example of impact](#example-of-cosmicqc-impact)
+5. 🚀 [Where do we go from here?](#whats-next)
+
+## Defining quality control of single cell morphology profiles
+
+> **Quality Control (QC)**  
+> The process of validating and enforcing a specific standard across all products.
```

Comment:
Consider adding the context of "scientific quality control" to help distinguish from industrial or business procedures which use the same language but may differ in focus.

Reply:
I am doing just general definitions here to show that this topic is universal, but thanks for the suggestion!